### DISCUSSION TITLE
Custom field for components and accessories

### DISCUSSION THREAD

**Discussion by EhfNina** [snipe/snipe-it#14453] [Q&A]
Hey Community,

I'm new in Snipe-IT. At the moment I want to test how I can inventory a smartphone and the installed SIM card in Snipe-IT.
I have created an asset for the smartphone with custom fields. Now I need a accessories or a component with custom fields for the SIM card, like PIN or callnumber.
But I can't find how to add custom fields to other objects except the asset object.

Can somebody help me please with this question? :-)

Many Thanks!

**Comment by CollinT-SeedIP** [comment]
Hi @EhfNina,

Unfortunately, custom fields for Accessories and consumables are not possible out of the box. I am also interested in this capability.

(edit) Here is the Issue tracking this request along with all associated/duplicate requests - [Custom Fields for Non-Assets (Licenses, Accessories, Consumables, Components, Users, Models, etc) #4261](https://github.com/snipe/snipe-it/issues/4261)

Thanks!
